cpp/data/zotero_cgi/qa.html

```diff
@@ -33,11 +33,11 @@ <h1>QA Settings</h1>
                 </tr>
             </thead>
             <tbody>
-                {LOOP tags, global_settings, journal_settings}
+                {LOOP tags, subfield_codes, global_settings, journal_settings}
```

Comment:
There should be a help text somewhere in this file that tells the user which checks are hardcoded (e.g. 001, 003, 007)

Reply:
There is now.